snap/local/hooks/cmd/configure/configure.go

```diff
@@ -86,146 +109,262 @@ func serviceType(name string) int {
 	}
 }
 
+func appendServices(startServices []string, newServices []string) []string {
+	for _, s := range newServices {
+		startServices = append(startServices, s)
+	}
+	return startServices
```

Comment:
this can also just be simplified to one append:

```suggestion
	return append(startServices, newServices...)
```